File: MODUL 3/Soal 5/PRAK205-2210817310008-Muh_Haekal_Barera.py
import math
a,b=input().split()
a=int(a)
b=int(b)
c=math.sqrt((b*b)-(a*a))
k=a+b+c
l=c*a/2
print("Alas = %d cm"%c)
print("Tinggi = %d cm"%a)
print("Keliling = %d cm"%k)
print("Luas = %d cm^2\n"%l)

a2=int(input())
b2=int(input())
c2=math.sqrt((b2*b2)-(a2*a2))
k2=a2+b2+c2
l2=c2*a2/2
print("Alas = %d cm"%c2)
print("Tinggi = %d cm"%a2)
print("Keliling = %d cm"%k2)
print("Luas = %d cm^2"%l2)

File: MODUL 3/Soal 5/PRAK205-2210817310008-Muh_Haekal_Barera.c
#include<stdio.h>

int main()
{
    for(int i=0; i<=1; i++){
    int s1,s2,s3,kel,luas;
    scanf("%d %d", &s1,&s2);
    s3 = sqrt((s2*s2)-(s1*s1));
    kel = s1+s2+s3;
    luas = s3*s1/2;
    printf("\nAlas = %d cm\n", s3);
    printf("Tinggi = %d cm\n", s1);
    printf("Keliling = %d cm\n", kel);
    printf("Luas = %d cm^2\n", luas);
    }
    return 0;
}
